data/quick-builds/separatist-alliance.json

```diff
@@ -92,7 +92,7 @@
           "id": "darkcourier",
           "upgrades": {
             "crew": ["countdooku", "generalgrievous"],
-            "tactical-relay": ["k2b4"],
+            "tacticalrelay": ["k2b4"],
```

Comment:
I changed this because xwd2 currently expects `tacticalrelay`. I see YASB2 outputs `tactical-relay` so perhaps xwd2 is wrong and it should change everywhere.